bluebutton-data-pipeline-rif-load/src/test/java/gov/hhs/cms/bluebutton/data/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -104,6 +104,7 @@ public void loadSampleU() {
 	 * Runs {@link RifLoader} against the
 	 * {@link StaticRifResourceGroup#SAMPLE_B} data.
 	 */
+	@Ignore
```

Comment:
Everything looks good to me.
Just one question, and this was probably mentioned elsewhere and I missed it, but is there a reason we are now Ignoring the loadSampleB and loadSyntheticData tests?

Reply:
They fail a lot of the time, I've noticed. We should probably address that in a separate PR, though, so I'd remove these `@Ignore`s from here.